.changeset/heavy-jobs-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
+---
```

Comment:
Let's patch i18n and myaccount(dependency of i18n) as well

Reply:
+ patch i18n as well

Reply:
Fixed